job_bundles/README.md

```diff
@@ -0,0 +1,315 @@
+    * `deadline:maxFailedTasksCount` - Value must be an integer, and is passed as the `maxFailedTasksCount` request parameter to the API.
+    * `deadline:maxRetriesPerTask` - Value must be an integer, and is passed as the `maxRetriesPerTask` request parameter to the API.
+
+While a standalone Job Template is viewed as a template from which Jobs can be created, a Job Bundle can be viewed as both
+a template from which to create Jobs and a representation of a specific Job. It represents a specific Job when the Job Bundle's parameter
+values file contains all of the Job's Parameter values that the Job is submitted with.
+
+For example, the [`blender_render` sample](https://github.com/aws-deadline/deadline-cloud-samples/tree/mainline/job_bundles/blender_render)
+has no parameter values file, and its Job Template defines Job Parameters, like `BlenderSceneFile` and `OutputDir`, that
+have no default values, so it must be used as a template from which Jobs are created. After we create a Job using this Job Bundle (with
+`deadline bundle gui-submit blender_render/` for example) a new Job Bundle is written to the job history directory and that Job Bundle has
+a `parameter_values.yaml` file that contains the values of all parameters we specified:
+
+```bash
+% cat ~/.deadline/job_history/\(default\)/2024-06/2024-06-20-01-JobBundle-Demo/parameter_values.yaml
+parameterValues:
+- name: deadline:targetTaskRunStatus
+  value: READY
+- name: deadline:maxFailedTasksCount
+  value: 10
+- name: deadline:maxRetriesPerTask
+  value: 5
+- name: deadline:priority
+  value: 75
+- name: BlenderSceneFile
+  value: /private/tmp/bundle_demo/bmw27_cpu.blend
+- name: Frames
+  value: 1-10
+- name: OutputDir
+  value: /private/tmp/bundle_demo/output
+- name: OutputPattern
+  value: output_####
+- name: Format
+  value: PNG
+- name: CondaPackages
+  value: blender
+- name: RezPackages
+  value: blender
+```
+
+We could recreate the exact same Job by submitting this new Job Bundle instead with: 
+
+```
+deadline bundle submit ~/.deadline/job_history/\(default\)/2024-06/2024-06-20-01-JobBundle-Demo/
+```
+
+Note: The submitted Job Bundle is saved to your "job history directory" and the location of that directory 
+can be found by running `deadline config get settings.job_history_dir`.
+
+### Elements - Asset References
+
+The `asset_references.yaml`/`asset_references.json` file in a Job Bundle is an interface for Deadline Cloud's
+[Job Attachments feature](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html)
+that lists input files and directories, as well as output directories for the Jobs that you
+submit. If you do not have all of your inputs and outputs for Job Attachments listed in this file
+then you can still select them for your Job during submission with the `deadline bundle gui-submit` CLI command.
+This file has no effect if you are not using Deadline Cloud's Job Attachments feature.
```

Comment:
I don't think we should state this, because it will stop being true the moment someone implements an Open Job Description tool that uses the file outside of job attachments.

Reply:
Sure